microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -721,8 +713,21 @@ Status:
 Controls whether to replay input events on the Timeline.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 109.
-*  This checkbox is present in Microsoft Edge Stable 107.
+*  This checkbox is present in Microsoft Edge Canary 111.
+*  This checkbox is present in Microsoft Edge Stable 108.
+
+
+<!-- ====================================================================== -->
+## View console.profile() results in the Performance panel for Node.js
+
+_new as of v109_
+
+Displays `console.profile()` results in the **Performance** tool for Node.js.
+<!-- todo: clarify -->
```

Comment:
@captainbrosset - any clarification? "View console.profile() results in the Performance panel for Node.js"

Reply:
I don't know anything about this, and reading the associated source code wasn't particularly helpful. We'll need to ask someone who works on the DevTools code to know for sure.

What I can tell is this:

`console.profile()` can be used in the console (or in your code) to start a JavaScript profile, and then you can use `console.profileEnd()` to terminate it. Once done, you can see your profile recording in the **JavaScript Profiler** tool.
This is a programmatic way to record JavaScript execution, which can be useful in certain perf debugging situations.

I don't think this has any relationship with the **Performance** tool though, that's confusing.

The other thing is that DevTools can optionally be used to debug processes running in Node.js. You need to start node in a special way so it exposes debugging info on a port. And then you can start DevTools from edge://inspect, to target that port.

So, presumably, this experiment makes whatever `console.profile` generates available to the **Performance** tool, also when DevTools is connected to Node.js.

Reply:
added text